### PR TITLE
Check dependencies added to Fauxton build

### PR DESCRIPTION
**Pull request by garrensmith**
run `npm run webpack:depcheck` to list all dependencies

### CODE REVIEW

**Comment by garrensmith** [comment]
My output when I run it

```
> fauxton@1.1.8 webpack:depcheck /Users/garren/dev/couchdb-fauxton
> node ./bin/depCheck.js


├── react-select@1.0.0-beta14
│   ../node_modules/react-select
├── jquery@2.2.4
│   ../node_modules/jquery
├── lodash@3.10.1
│   ../node_modules/lodash
├── backbone@1.3.3
│   ../node_modules/backbone
├── d3@3.5.17
│   ../node_modules/d3
├── moment@2.14.1
│   ../node_modules/moment
├── backbone.layoutmanager@0.9.7
│   ../node_modules/backbone.layoutmanager
├─┬ react@15.0.2
│ │ ../node_modules/react
│ └── fbjs@0.8.3
│     ../node_modules/react/node_modules/fbjs
├── object-assign@4.1.0
│   ../node_modules/object-assign
├── react-dom@15.0.2
│   ../node_modules/react-dom
├── flux@2.1.1
│   ../node_modules/flux
├── fbjs@0.1.0-alpha.7
│   ../node_modules/fbjs
├── css-loader@0.23.1
│   ../node_modules/css-loader
├── style-loader@0.13.1
│   ../node_modules/style-loader
├── zeroclipboard@2.2.0
│   ../node_modules/zeroclipboard
├─┬ react-bootstrap@0.28.5
│ │ ../node_modules/react-bootstrap
│ ├── babel-runtime@5.8.38
│ │   ../node_modules/react-bootstrap/node_modules/babel-runtime
│ └── core-js@1.2.7
│     ../node_modules/react-bootstrap/node_modules/core-js
├── react-prop-types@0.3.2
│   ../node_modules/react-prop-types
├── invariant@2.2.1
│   ../node_modules/invariant
├── warning@2.1.0
│   ../node_modules/warning
├── classnames@2.2.5
│   ../node_modules/classnames
├── dom-helpers@2.4.0
│   ../node_modules/dom-helpers
├─┬ react-overlays@0.6.6
│ │ ../node_modules/react-overlays
│ ├── warning@3.0.0
│ │   ../node_modules/react-overlays/node_modules/warning
│ └── react-prop-types@0.4.0
│     ../node_modules/react-overlays/node_modules/react-prop-types
├── keycode@2.1.4
│   ../node_modules/keycode
├── lodash-compat@3.10.2
│   ../node_modules/lodash-compat
├── uncontrollable@3.3.1
│   ../node_modules/uncontrollable
├── velocity-animate@1.2.3
│   ../node_modules/velocity-animate
├── react-addons-css-transition-group@15.0.2
│   ../node_modules/react-addons-css-transition-group
├── brace@0.7.0
│   ../node_modules/brace
├── w3c-blob@0.0.1
│   ../node_modules/w3c-blob
├── velocity-react@1.1.5
│   ../node_modules/velocity-react
├── react-addons-transition-group@15.3.0
│   ../node_modules/react-addons-transition-group
├── react-input-autosize@1.1.0
│   ../node_modules/react-input-autosize
├── blacklist@1.1.4
│   ../node_modules/blacklist
├── visualizeRevTree@0.0.1
│   ../node_modules/visualizeRevTree
├── pouchdb@5.4.5
│   ../node_modules/pouchdb
├── process@0.11.8
│   ../node_modules/process
├── js-extend@0.0.1
│   ../node_modules/js-extend
├── debug@2.2.0
│   ../node_modules/debug
├── ms@0.7.1
│   ../node_modules/ms
├── inherits@2.0.1
│   ../node_modules/inherits
├── lie@3.0.4
│   ../node_modules/lie
├── immediate@3.0.6
│   ../node_modules/immediate
├── pouchdb-collections@1.0.1
│   ../node_modules/pouchdb-collections
├── argsarray@0.0.1
│   ../node_modules/argsarray
├── events@1.1.1
│   ../node_modules/events
├── scope-eval@0.0.3
│   ../node_modules/scope-eval
├── spark-md5@2.0.2
│   ../node_modules/spark-md5
├── vuvuzela@1.0.3
│   ../node_modules/vuvuzela
├── es6-promise-pool@2.4.2
│   ../node_modules/es6-promise-pool
├── pouchdb-collate@1.2.0
│   ../node_modules/pouchdb-collate
├─┬ jsondiffpatch@0.1.43
│ │ ../node_modules/jsondiffpatch
│ ├── chalk@0.5.1
│ │   ../node_modules/jsondiffpatch/node_modules/chalk
│ ├── ansi-styles@1.1.0
│ │   ../node_modules/jsondiffpatch/node_modules/ansi-styles
│ ├── strip-ansi@0.3.0
│ │   ../node_modules/jsondiffpatch/node_modules/strip-ansi
│ ├── ansi-regex@0.2.1
│ │   ../node_modules/jsondiffpatch/node_modules/ansi-regex
│ ├── has-ansi@0.1.0
│ │   ../node_modules/jsondiffpatch/node_modules/has-ansi
│ └── supports-color@0.2.0
│     ../node_modules/jsondiffpatch/node_modules/supports-color
└── escape-string-regexp@1.0.5
    ../node_modules/escape-string-regexp
```

**Comment by robertkowalski** [comment]
+1 after adding a license header

File: webpack.config.release.js [hunk 1]
```diff
@@ -11,6 +11,8 @@
 // the License.
 var webpack = require('webpack');
 var path = require('path');
+var depsWebpackPlugin = require('deps-webpack-plugin');
+var path = require('path');
```
Comment (robertkowalski) [inline webpack.config.release.js]: path is not required here two times

File: bin/depCheck.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var webpack = require("webpack");
```
Comment (robertkowalski) [inline bin/depCheck.js]: const and singel quotes

File: bin/depCheck.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+const config = require('../webpack.config.release');
+const helper = require('webpack-dependency-tree');
+// returns a Compiler instance
+webpack(config, function(err, stats) {
```
Comment (robertkowalski) [inline bin/depCheck.js]: () => {}

**Comment by robertkowalski** [comment]
started to build a tool around this...

**Comment by garrensmith** [comment]
I'm closing this since @robertkowalski has some nice plugins to do this for us now.

**Pull request closed without merging**